controllers/gslb_controller.go

```diff
@@ -154,13 +154,15 @@ func (r *GslbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 	// == handle delegated zone in Edge DNS
 	err = r.DNSProvider.CreateZoneDelegationForExternalDNS(gslb)
 	if err != nil {
-		return result.RequeueError(err)
+		log.Err(err).Msgf("Create zone delegation")
```

Comment:
- `Msg()` can be used instead of `Msgf()` here
- It will look like `Create zone delegation error="error message aaaaa"` only in case of simplified logging format. With json logging format, error context will be provided as separate field, so the message will not look obvious for user in the log or log aggregation solution.
 
```suggestion
		log.Err(err).Msg("Create zone delegation error")
```

Reply:
good point, thx 👍